pkg/core/app/util.go

```diff
@@ -16,7 +16,14 @@ import (
 	"go.uber.org/zap"
 )
 
-func findComposeFile() string {
+func findComposeFile(cmdArgs []string) string {
```

Comment:
it would be better if we pass the command rather than cmdArgs in the function param.